android/src/main/java/com/twiliorn/library/Utils.java

```diff
@@ -105,7 +105,7 @@ private static byte[] rotateJPEGByteArray(
             Bitmap sourceBitmap = BitmapFactory.decodeByteArray(data, 0, data.length);
             Bitmap rotatedBitmap = rotateBitmap(sourceBitmap, rotation);
             ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
-            // rotatedBitmap.compress(Bitmap.CompressFormat.JPEG, 100, byteArrayOutputStream);
+            rotatedBitmap.compress(Bitmap.CompressFormat.JPEG, 100, byteArrayOutputStream);
```

Comment:
This line was commented out as part of ST-6199 but should not have been, as it results in a 0-byte buffer being returned.